include/AIToolbox/MDP/TypeTraitsEigen.hpp

```diff
@@ -0,0 +1,87 @@
+#ifndef AI_TOOLBOX_MDP_TYPE_TRAITS_EIGEN_HEADER_FILE
```

Comment:
I don't like this file as well. The library supports Eigen, and it's a chore for me to maintain more files than what I have just so that some methods can be more freely taken out of the library as stand-alone.

Reply:
See above.